app/api/routers/breeze_buddy/smallwebrtc/__init__.py

```
@@ -0,0 +1,37 @@
+from app.schemas import UserInfo
+
+from .handlers import smallwebrtc_offer_handler, smallwebrtc_patch_handler
+
+router = APIRouter()
+
+
+@router.post("/smallwebrtc/offer")
+async def smallwebrtc_offer(
+    body: Dict[str, Any],
+    request: Request,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)